app/serializers/collection.js

```diff
@@ -1,5 +1,8 @@
 import DS from 'ember-data';
 
 export default DS.JSONSerializer.extend({
-    primaryKey: 'collectionId'
+    primaryKey: 'collectionId',
```

Comment:
I think you can remove the primaryKey override.

By default Ember-Data will use the key `id` as primary key. The `id` key should be added to all responses from the API. See https://guides.emberjs.com/v2.6.0/models/customizing-serializers/#toc_ids for more details